libs/gltfio/src/MaterialGenerator.cpp

```diff
@@ -160,6 +157,26 @@ static std::string shaderFromKey(const MaterialKey& config, const UvMap& uvmap)
     }
 
     shader += "}\n";
+
+    auto replaceAll = [&shader](const std::string& from, const std::string& to) {
+        size_t pos = shader.find(from);
+        for (; pos != std::string::npos; pos = shader.find(from, pos)) {
+            shader.replace(pos, from.length(), to);
+        }
+    };
+
+    const auto normalUV = std::to_string(uvmap[config.normalUV] - 1);
+    const auto baseColorUV = std::to_string(uvmap[config.baseColorUV] - 1);
+    const auto metallicRoughnessUV = std::to_string(uvmap[config.metallicRoughnessUV] - 1);
+    const auto emissiveUV = std::to_string(uvmap[config.emissiveUV] - 1);
+    const auto aoUV = std::to_string(uvmap[config.aoUV] - 1);
+
+    replaceAll("{normal}", normalUV);
```

Comment:
Nit, `${xxx}` would match string interpolation in many other languages (like Kotlin)

Reply:
Went ahead and changed this, then realized that I probably had Python f-strings and Nim in the back of my mind.  :)